src/BODSDataExtractor/extractor.py

```diff
@@ -78,23 +78,24 @@ def check_api_response(self, response,j1):
         #if response is less than 4, no data has been pulled
         
         if len(j1)<4:
-            print("_____________________________________")
-            print("Error")
+            message=""

```

Comment:
is this blank message necessary?

Reply:
Will initialise this at the beginning of the function